app/controllers/register_controller.rb

```diff
@@ -0,0 +1,25 @@
+class RegisterController < ApplicationController
+
+  def new
+  end
+
+  def create
+    @user = User.new(user_params)
+    if User.exists?(email: user_params[:email])
+      flash[:error] = "Email already in use. Try a different email."
+      render :new
+    elsif @user.save
```

Comment:
So some additional changes we can touch up today. Instead of just `@user.save` We can get ride of the confirm_pass column entirely and have it double check the passwords are matching here. `elsif params[:confirm_pass] == params[:password] && @user.save`

Reply:
I also agree with this